### PR TITLE
Stun Baton Rebalance & Energy Baton Addition

### PR DESCRIPTION
**Pull request by BramvanZijp**
## About The Pull Request

This Pull Request makes the following changes:
The stun baton loses its ability knockdown a target after 2 seconds.
A new energy baton is added, which can be researched and then created using an upgrade kit.
The energy baton has 3 settings: Off, Stun, Kill.
On stun, it is the current stun baton functionality, including the delayed knockdown, which is now restricted to this item.
On kill, it deals heavy (47.5) burn damage, and does a short but immediate knockdown, at the cost of double the charges. (This respects the batons 2.5 hit delay, and for structures the damage is heavily reduced, so the DPS is much lower then it sounds, 19 to be exact.)
The Head of Security gets an energy baton in their locker roundstart.

image

This PR should probably be test merged, and I am open to feedback on both the functionality and the sprite of the energy baton.

## Why It's Good For The Game

I have seen many people wanting nerfs to the knockdown of stun batons, and even a pr removing them entirely. I believe this is a bit excessive. With this PR, the ability to knockdown by security without a telebaton or police baton (Which should be removed from cargo but that is its own PR) is restricted to a high level tech and the officer is required to actually upgrade their baton. Furthermore, the kill setting adds more flavor to the weapon, gives security a lethal melee option that is specific to them instead of "just grab a welder or cleaver" that can also knockdown like a police or telebaton, albeit requiring reason for lethal escalation and taking further charges. 
In short, this means that the standard baton is nerfed, with a later tech item being added to make up for that.

## Changelog

:cl: BramvanZijp
add: Added a new Energy Baton that is issued to the HoS and can be researched by science. Alongside a stun setting that can knockdown after a delay, it has a new kill setting that deals heavy burn damage and an immediate knockdown.
balance: Standard Stun Batons no longer knock down their target after a small time delay.
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
Get any stun baton nerf cleared with me.

image

I told you several times over Discord that we do not want a variant of the stun baton with explicit "kill" modes--the stun baton itself is already fairly lethal and doing stuff like this only encourages a more lethal sec which I don't want any more than we already have.
